website/docs/tutorials/02-installation.md

```diff
@@ -32,6 +32,12 @@ To upgrade Protostar, run:
 $ protostar upgrade
 ```
 
+# Protostar releases
+The Protostar team tries to release a new minor version every 2 weeks.
+Each minor release may include new features, enhancements, bug fixes, depreciation warnings, and breaking changes.
+Breaking changes can be found in the changelog.
+Protostar shows a link to the changelog when a new release is detected.
```

Comment:
```suggestion

Protostar follows a biweekly release schedule.
Each release may include new features, enhancements, bug fixes, deprecations and breaking changes.
Consult [release notes](https://github.com/software-mansion/protostar/releases) for detailed information about each release.
Protostar prints a link to new release before upgrading.
```

What I had comments about:
1. I would try to avoid referring to `Protostar team`
2. I still find saying what major, minor or patch does because it's irrelevant for end user.
3. Added a link to release notes of all releases.